src/client/login/components/LoginForm.tsx

```diff
@@ -122,7 +132,6 @@ const LoginForm = ({
           <TextButton
             className={classes.secondaryButton}
             href={i18next.t('general.links.faq')}
-            target="_blank"
```

Comment:
?

Reply:
I shifted it to inside the TextButton since it is only used once